OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -1637,6 +1637,11 @@ public void onSuccess() {
       public void onFailure(OneSignal.EmailUpdateError error) {
          emailFiredFailure = error;
       }
+
+      @Override
```

Comment:
TravisCI is failing due to a Java compile error due to this interface method no longer existing.